sky/server/common.py

```diff
@@ -57,12 +58,44 @@
 # (e.g. in high contention env) and we will exit eagerly if server exit.
 WAIT_APISERVER_START_TIMEOUT_SEC = 60
 
-SKY_API_VERSION_WARNING = (
-    f'{colorama.Fore.YELLOW}SkyPilot API server is too old: '
-    f'v{{server_version}} (client version is v{{client_version}}). '
-    'Please restart the SkyPilot API server with: '
+VERSION_INFO = (
```

Comment:
Might be better to keep these in constants.py?

Reply:
`common` would be better since we've made these private